### PR TITLE
feat(casperf/lvol): add lvol uri type

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This allows us to run casperf against an lvol, example: sudo casperf
"lvol:///lvol-1?size=220GiB&lvs=lvs:///pool-1?mode=purge&disk=aio:///dev/sda" -b 128ki -q 1

### CODE REVIEW

File: io-engine/src/bdev/lvs.rs [hunk 1]
```diff
@@ -0,0 +1,279 @@
+//! Allows creation of an lvs lvol through a URI specification rather than gRPC
+//! or direct function call. This is not intended for the usual product
+//! operation but for testing and benchmarking.
+//!
+//! # Uri
+//! lvol:///$name?size=$size&lvs=$lvs
```
Comment (hrudaya21) [inline io-engine/src/bdev/lvs.rs:6]: How this uri is used in io-engine-client CLI ? 
`./io-engine-client pool create lvol:///$name?size=$size&lvs=$lvs`

**Review by hrudaya21 — APPROVED** [review]

File: io-engine/src/bdev/lvs.rs (hunk 1 above)
Comment (hrudaya21) [reply io-engine/src/bdev/lvs.rs:6]: Sorry.. understood.. It is part of PR description :)

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**